src/main/resources/com/mathworks/ci/TestResultsViewAction/index.jelly

```diff
@@ -0,0 +1,217 @@
+                                        <l:icon class="symbol-checkmark-circle-outline plugin-ionicons-api icon-lg jenkins-!-success-color" tooltip="Passed" />
+                                    </j:if>
+                                    <j:if test="${testFile.status.contains('Failed')}">
+                                        <l:icon class="symbol-close-circle-outline plugin-ionicons-api icon-lg jenkins-!-error-color" tooltip="Failed" />
+                                    </j:if>
+                                    <j:if test="${testFile.status.contains('Incomplete')}">
+                                        <l:icon class="symbol-alert-circle-outline plugin-ionicons-api icon-lg jenkins-!-warning-color" tooltip="Incomplete" />
+                                    </j:if>
+                                    <j:if test="${testFile.status.contains('NotRun')}">
+                                        <l:icon class="symbol-remove-circle-outline plugin-ionicons-api icon-lg" tooltip="Not Run" />
+                                    </j:if>
+                                    <div style="display: inline;" tooltip="${testFile.filePath}">
+                                        <j:whitespace trim="false"> ${testFile.name} </j:whitespace> 
+                                    </div>
+
+                                    <a id="plus-${testFile.name}-${testFile.id}" onclick="toggleVisibilty('${testFile.name}-${testFile.id}');" style="display: inline;">
+                                        <l:icon class="symbol-add plugin-ionicons-api icon-sm" tooltip="Show tests" />
+                                    </a>
+                                    <a id="minus-${testFile.name}-${testFile.id}" onclick="toggleVisibilty('${testFile.name}-${testFile.id}');" style="display: none">
+                                        <l:icon class="symbol-remove plugin-ionicons-api icon-sm" tooltip="Hide tests" />
+                                    </a>
+
+                                <div id="testCases-${testFile.name}-${testFile.id}" style="display: none;">
+                                    <br />
+                                    <br />
+                                    <table class="jenkins-table">
+                                        <thead>
+                                            <tr>
+                                                <th class="pane-header" style="width: max-content;">
+                                                    Test Case
+                                                </th>
+                                                <th class="pane-header" style="width: 100%;">
+                                                    Diagnostics
+                                                </th>
+                                                <th class="pane-header" style="text-align: right; text-wrap:nowrap;">
+                                                    Duration (seconds) = ${testFile.duration}

```

Comment:
= or :?

Reply:
Since colon is something already associated with showing time (like HH:MM:SS) in some sense, to avoid confusion I chose =

Reply:
I think a single colon would work; however, I agree that something like (HH:MM:SS): would be confusing. The equals sign might be misinterpreted as an assignment operator :)

Reply:
Sure,  if that doesn't cause any confusion!